security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/function/oci-datasafe-audit-to-logging/build-fn.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+. ./set-local-vars.sh
+
+mkdir dist
+rsync -aC --exclude node_modules --exclude dist --exclude '.git' --exclude '*.sh' --exclude test --exclude '.gitignore' . ./dist
```

Comment:
What is this for? Looks dangerous to me. I would run this.

Reply:
removed from script